### PR TITLE
feat(Button): allow including loading placeholder

### PR DESCRIPTION
**Pull request by MkMan**
More than half this PR changes are snapshot files 😏 Please only review the 3 buttons inside the Button folder 🙏 

https://user-images.githubusercontent.com/33766083/161674877-c9689dd7-3e7d-4ab0-a9db-ddaddd21ab08.mov

### CODE REVIEW

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -17,12 +18,14 @@ const Button = forwardRef<HTMLButtonElement, ButtonProps>(function ButtonRef(
     isAlternate = false,
     isEnabled = true,
     isInline = false,
+    isLoading = false,
+    loadingScreenReaderText,
```
Comment (cyrusxzw) [inline src/components/Button/Button.tsx:22]: Nice to have `loadingScreenReaderText`.

**Comment by cyrusxzw** [comment]
What if button background color is set to dark, 3 dots may not clear due to contrast.

**Comment by MkMan** [comment]
> What if button background color is set to dark, 3 dots may not clear due to contrast.

Great pick up 👌 Let me look into it

**Comment by MkMan** [comment]
> What if button background color is set to dark, 3 dots may not clear due to contrast.

I added logic to change the `<Loading />` theme based on the button props. Thanks for alerting me 🙏

**Review by cyrusxzw — APPROVED** [review]

**Pull request merged**